src/library/styles/components/_input-radio.scss

```diff
@@ -1,21 +1,21 @@
 $outer-circle-diameter: $ffs-body-assertive;
 $inner-circle-diameter: space(md);
 
-.bsds-input-radio {
+.bsds-input-radio-control {
   display: flex;
 }
 
 .bsds-fieldset {
-  > .bsds-input {
+  > .bsds-field {
     &:not(:first-child) {
-      .bsds-input-radio {
+      .bsds-input-radio-control {
         margin-top: space(2x);
       }
     }
   }
 }
```

Comment:
Do we want to abstract this to a shared place since the spacing/code is the same as checkbox?

Reply:
Do we want to do the same with focus and invalid? Anything else?